app/shell-window/ui/prompts/fullscreen.js

```diff
@@ -0,0 +1,37 @@
+import * as yo from 'yo-yo'
+// import { ??? } from 'electron'
+import * as promptbar from '../promptbar'
+
+// constants
+// =
+
+const FULLSCREEN_WARNING_TIME = 2000
+
+// exported api
+// =
+
+export default function () {
+  return {
+    type: 'fullscreen',
+    render: () => { 
+      const onClickClose = () => {
+        console.log('sending leave-full-screen')
+        // ???.send('window-event', 'leave-full-screen')
```

Comment:
This is the only bit not working - I want to send `window-event` of type `leave-full-screen` here, but don't know how to do that yet